service/waterfall.go

```diff
@@ -637,6 +637,20 @@ func (uis *UIServer) waterfallPage(w http.ResponseWriter, r *http.Request) {
 	}{newUILink, uis.Settings.Jira.Host, uis.GetCommonViewData(w, r, false, true)}, "base", "waterfall.html", "base_angular.html", "menu.html")
 }
 
+// Create and return a redirect to the spruce waterfall page. If the user is opted in to the new UI,
+func (uis *UIServer) waterfallRedirect(w http.ResponseWriter, r *http.Request) {
```

Comment:
Should we call this mainlineCommitsRedirect or something since we don't call the new page the waterfall?